SCF-Session-8-Assignment/src/question/Snake.java

```diff
@@ -0,0 +1,28 @@
+package question;
+
+/**
+ * This class is for Snake type animals
+ * @author Jugal
+ * Dated 07/24/2019
+ */
+public class Snake extends Reptile {
+	

```

Comment:
Declare some different properties of Snake class and initialize them in Snake class constructor and initialize Reptile and Animal class property using super().

Reply:
Done